apollo-router-core/src/error.rs

```diff
@@ -132,6 +129,54 @@ pub struct Error {
     pub extensions: Object,
 }
 
+/// temporary structure to help deserializing errors
```

Comment:
:yarn: Either it's a doc comment and it is properly written and ready to be in the doc. Either it's a comment and it can have typos or not be perfect. I know here the struct is private but if it becomes public, this comment will turn into a doc comment too easily and without review.

```suggestion
// temporary structure to help deserializing errors
```

Reply:
right, that makes more sense as a local code comment